temoa/model_checking/unit_checking/screener.py

```
@@ -0,0 +1,336 @@
+    else:
+        errors_test2 = True
+
+    report_entries.append('\n')
+    return not errors_test2
+
+
+def _check_efficiency_table(
+    conn: sqlite3.Connection,
+    report_entries: list[str],
+    comm_units: dict[str, Any],
+    c2a_units: dict[str, Any],
```

Ruff (0.14.7)

Unused function argument: `c2a_units`

(ARG001)